samples/SmokeTest/src/main/java/com/azure/CosmosDB.java

```diff
@@ -0,0 +1,127 @@
+package com.azure;
```

Comment:
These samples should live in their respective samples directories. (ie. $/sdk/cosmos/microsoft-azure-cosmos/src/samples/java). There are guidelines here:
https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html#general-documentation-code-samples

That way, our code style tools will run on each sample.

Reply:
Hey @conniey, 

The purpose of these smoke tests is to validate that we can load all relevant libraries into a single process and that the libraries can be used to interact with live services. To keep complexity down we have created arbitrary samples that perform specific scenarios (across all 4 languages). 

Eventually we hope to incorporate samples from their respective locations in the library code. For the purpose of getting a functioning vertical slice quickly it made sense to keep the scenarios confined to a more central location than modifying some subset of the existing examples. 

Is it ok to keep the code organized in this way?

Reply:
I'd defer this decision to @JonathanGiles or @joshfree. Is the only team using this the eng team?

Reply:
At this point we are the only team using this. Perhaps the name "samples" is a distraction given our goals.

Reply:
I'm ok with having this centralised like this, but I agree that `/samples` is not the right place for it. If we had more cross-cutting samples, having a top-level sample directory would make sense. At this stage, I would suggest `/eng/smoke-tests` is a better location for this code.